flows/definition/localization.go

```diff
@@ -11,35 +11,21 @@ type itemTranslations map[string][]string
 // languageTranslations map a node uuid to item_translations - say "node1-asdf" to { "text": "je suis francais!" }
 type languageTranslations map[flows.UUID]itemTranslations
 
-func (t *languageTranslations) GetTranslations(uuid flows.UUID, key string, backdown []string) []string {
+func (t *languageTranslations) GetTextArray(uuid flows.UUID, key string) ([]string, bool) {
```

Comment:
Is it useful to have a distinction between `nil` here? (which I'm assuming is the purpose of the bool return value?) Just wondering when a caller would ever act differently when receiving a nil value and true vs a nil value and false. (if never, then lets just return a single value and nil when not found)